src/main/java/util/TokenFromMSAL4J.java

```diff
@@ -0,0 +1,113 @@
+package util;
+
+import com.microsoft.aad.msal4j.*;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Set;
+
+public class TokenFromMSAL4J {
+    private static Loader loader = new LocalLoaderImpl();
+    private static String authority;
+    private static Set<String> scope;
+    private static String clientId;
+    private static String username;
+    private static String password;
+    public static  String token;
+    public static void main(String args[]) throws Exception {}
+        public static String createToken() throws Exception {
+        setUpSampleData();
+
+        PublicClientApplication pca = PublicClientApplication.builder(clientId)
+                .authority(authority)
+                .build();
+
+        //Get list of accounts from the application's token cache, and search them for the configured username
+        //getAccounts() will be empty on this first call, as accounts are added to the cache when acquiring a token
+        Set<IAccount> accountsInCache = pca.getAccounts().join();
+        IAccount account = getAccountByUsername(accountsInCache, username);
+
+        //Attempt to acquire token when user's account is not in the application's token cache
+        IAuthenticationResult result = acquireTokenUsernamePassword(pca, scope, account, username, password);
+//        System.out.println("Account username: " + result.account().username());
+//        System.out.println("Access token:     " + result.accessToken());
+//        System.out.println("Id token:         " + result.idToken());
+//        System.out.println();
+        token=result.idToken();
+        accountsInCache = pca.getAccounts().join();
+        account = getAccountByUsername(accountsInCache, username);
+
+        //Attempt to acquire token again, now that the user's account and a token are in the application's token cache
+        result = acquireTokenUsernamePassword(pca, scope, account, username, password);
+//        System.out.println("Account username: " + result.account().username());
+//        System.out.println("Access token:     " + result.accessToken());
+//        System.out.println("Id token:         " + result.idToken());
+        token=result.idToken();
```

Comment:
Remove this assignment, return token directly

Reply:
done